PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/View Controllers/ViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  ViewController.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Alex Paul on 12/7/18.
+//  Copyright © 2018 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class userViewController: UIViewController,UITableViewDelegate,UITableViewDataSource {
+    @IBOutlet weak var userTableView: UITableView!
+
+
+    @IBOutlet weak var userSearchBar: UISearchBar!
+    
+ 
```

Comment:
Seems like you wanted to implement the search functionality, but didn't quite get it. Let me know if you need help reviewing it at some point.